### PR TITLE
Add serbian language

### PR DESCRIPTION
**Pull request by ivkeapp**
### Context
Let's all contribute

### Types of changes

- [x] Added Serbian language

### Affected project(s):
- [x] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [x] My change requires a change to the documentation.

### CODE REVIEW

**Review by Cookie994 — APPROVED** [review]
Translation looking good! ^^

**Comment by AMBudnik** [comment]
Hi @ivkeapp 

Thank you for sharing the translation! Could you also sign [our CLA](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1) and change the branch to `develop`?

**Comment by ivkeapp** [comment]
> Hi @ivkeapp
> 
> Thank you for sharing the translation! Could you also sign [our CLA](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1) and change the branch to `develop`?

Hello, gladly. :)

**Comment by ivkeapp** [comment]
> Translation looking good! ^^

Thanks

**Review by nemanjadvlp — APPROVED** [review]
Good translation

**Comment by AMBudnik** [comment]
Thank you! I will pass the pull request for the code review.

**Comment by ivkeapp** [comment]
> Thank you! I will pass the pull request for the code review.

Thank you 👍

File: .changelogs/9419.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
[the remark this answers is not in the document]
Comment (ivkeapp) [reply .changelogs/9419.json:1]: Sure.

[the remark this answers is not in the document]
Comment (ivkeapp) [reply .changelogs/9419.json:1]: Hope I did it right @budnix

**Review by budnix — CHANGES_REQUESTED** [review]
Thanks! We are almost there. Just some nitpick changes to be done.
 * Revert `.changelogs/9419.json` changelog file as it belongs to another PR (you've already added correct one - 9469.json);
 * Our linter requires new lines at the end of each source code file. Can you add that?
   * https://github.com/handsontable/handsontable/pull/9469/files#diff-cbe5e6112b4f48d06305fb52e3d6a41c30f28cd25c11761fc6d3d49e240f953dR7
   * https://github.com/handsontable/handsontable/pull/9469/files#diff-c3872efa05482bf7db8e0a315d98d7c8dd95b2eb869c8d9d49b6dfe4ce5bb707R102

**Comment by budnix** [comment]
@ivkeapp Thanks!

**Comment by budnix** [comment]
@kirszenbaum May I ask you to prepare the PR with docs changes about the new language?

**Comment by AMBudnik** [comment]
Hi @ivkeapp I'm more than happy to announce that we just released Handsontable v12.1.0 with the Serbian translation. Thanks again for the contribution!

**Comment by adrianszymanski89** [comment]
Hi @ivkeapp. We just announced an introduction of a new functionality called `Copy with headers`. Could you help us to translate three short lines for the context menu?

Here you can find more information about it: https://github.com/handsontable/handsontable/issues/10112

**Comment by ivkeapp** [comment]
Hi, I have done a translation to Serbian :)
    On Friday, November 18, 2022 at 01:41:18 PM GMT+1, adrianszymanski89 ***@***.***> wrote:  
 
 


Hi @ivkeapp. We just announced an introduction of a new functionality called Copy with headers. Could you help us to translate three short lines for the context menu?

Here you can find more information about it: #10112

**Pull request merged**